.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
get this out of the repo and add an entry in a git ignore file

Reply:
removed